smsd/core.h

```diff
@@ -110,6 +110,7 @@ struct _GSM_SMSDConfig {
 	GSM_SMSC	SMSC, SMSCCache;
 	const char	*skipsmscnumber;
 	int		IgnoredMessages;
+	gboolean 	SkipMessage[GSM_MAX_MULTI_SMS];
```

Comment:
This should be initialized somewhere to avoid possible problems with backends not setting the flag (eg. files).